src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.print("Figure: circle, area "
+                + calculateArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. Here, `drawFigure` should be part of a separate interface, as drawing and calculating area are conceptually different. Also, consider renaming `drawFigure` to `draw` to be consistent with the single responsibility principle.